infrastructure/simulation-tool/src/operations/transfer.ts

```diff
@@ -0,0 +1,141 @@
+import { BigNumber } from 'ethers';
+import { Wallet as RollupWallet, Transaction, utils } from '@rsksmart/rif-rollup-js-sdk/';
+import config from '../utils/config.utils';
+import { getRandomBigNumber } from '../utils/number.utils';
+import { sleep } from '@rsksmart/rif-rollup-js-sdk/build/utils';
+import { Address, TransactionReceipt } from '@rsksmart/rif-rollup-js-sdk/build/types';
+import { getRandomElement } from './common';
+
+type PreparedTransfer = Omit<Parameters<RollupWallet['syncTransfer']>[number], 'amount'> & {
+    amount: BigNumber;
```

Comment:
@jurajpiar  We omit amount and then add it manually only to change its type from `BigNumberish` to `BigNumber`, don't we? If so, are we sure that it's worth it?